string label_mask
string label_mask_prob
int64 x_start
int64 x_end
int64 y_start
int64 y_end
string label_pose
string label_pose_prob
float32 x_pose
float32 y_pose


